ramls/accounts.raml

```diff
@@ -108,3 +110,31 @@ resourceTypes:
             body:
               text/plain:
                 example: "Internal server error"
+    /pay:
+      post:
+        is: [validate]
```

Comment:
Do we need it? It seems like we use java validation only

Reply:
We use java-validation just for one field. RMB should validate the rest.